### DISCUSSION TITLE
v1.5.6.2

### DISCUSSION THREAD

**Discussion by krajanos** [fedorovvl/tso_client#257] [General]
From realease notes:

> [army] window removed

How can we load army now? For example on home island.

**Comment by fedorovvl** [comment]
use battle window, now it includes all army functions (load, unload, send to AT, etc)

**Comment by JohnRo79** [comment]
old saved templates won't work with this new feature
are you planning to change it soon or will it stay like this?

**Comment by Mazimus332** [comment]
My saved army templates from 1.5.6.1 worked with 1.5.6.2 mundane warfare.
Old mundane warfare templates with movement and attack also worked, you will only need to re-save templates but now with army+move+attack, but all should essentially work, it only seems that very old templates don't work.

**Comment by krajanos** [comment]
Old templates works but they don't have information about generals army.
I used to load army with one script and move/attack with second script.

Old templates (Boring warfare/Battle window) can be fixed in two ways:

1. Load army to generals, load template and save it
2. Edit existing templates with notepad, add for example:
`"army": {
   "Militia": 230
  }`

**Comment by Opferlamm** [comment]
Is it possible to send Generals from Starmenue to Homeisland, then fill and send to AT?

**Reply by fedorovvl** [reply]
sure.. just make a template for that =)

**Reply by Opferlamm** [reply]
Thanks for fast answer, but i tried it again and again.
But i see no option to send generals from starmenue to homeisland (in own homezone)
https://prnt.sc/B13bWAcykxqJ
it is only possible to send generals to starmenu :-(

**Reply by Mazimus332** [reply]
First place generals at Home island, where you want always to place them.
My advice: try to avoid using spots that random generals land on when arriving from adventure, use the "free spots" that are left.

![Screenshot_467](https://user-images.githubusercontent.com/116684854/234214176-9d83e532-462b-4fa8-992e-24f30f2e70d4.png)

In my case, I only use Quartermaster's to transport units, so I send every other general to star.
Then, when you have all your Quartermasters/Generals positioned where you want them, save template "Homeisland.txt" selecting every single general, when you load this template and click "Move" it will move your quartermasters to the defined spots and move all your combat generals to star menu.

![Screenshot_468](https://user-images.githubusercontent.com/116684854/234215129-067f6a53-bd8a-44a8-bd81-e5db9de5f3a4.png)

**Reply by Opferlamm** [reply]
Hi Mazimus332, thanks for reply, but
move, fill and sent to star menue, that all is not the problem.
My wish is sent needed generals from starmenu to home island.
eg:
all my generals are in star on my homezone. for The_Nords i need Nusala Anslem Vargus... and only these generals should land on my island to be filled.

**Reply by Mazimus332** [reply]
Read carefully again, what you are requesting can be done, and I already answered your question.

You can use a template to land on home island only on predefined spots that you have chosen before saving a template (with Move button), load them their army and then send them with the same template.

If what you are requesting automatic "random" land on home island if free positions left, I highly doubt that will be done. 

I suggest you joining DE discord server if you want more general support on your native language https://discord.gg/Wxhsc4bBCP

**Reply by Opferlamm** [reply]
It works, my son set it up for his 68 year old father. Thanks for the help
